learning-ojs/en/settings-workflow.md

```diff
@@ -62,6 +70,10 @@ Use this field to add information for your authors that will be useful during th
 
 ## Review
 
+This PKP School video demonstrates how to configure the Review process in OJS. To watch other videos in this series, visit [PKP’s YouTube channel](https://www.youtube.com/playlist?list=PLg358gdRUrDVTXpuGXiMgETgnIouWoWaY).
+
+{% include video.html id="1adbIbQM6E" provider="youtube" title="Video of how to configure the Review process in OJS"%}
```

Comment:
This video doesn't seem to load - perhaps it was taken down for further editing?